src/ZoneServer/Buffs/Buff.cs

```diff
@@ -267,6 +267,17 @@ internal void ExtendDuration()
 				this.NextUpdateTime = DateTime.Now.Add(this.Data.UpdateTime);
 		}
 
+		/// <summary>
+		/// Increases the buff's duration by a given amount.
+		/// </summary>
+		internal void IncreaseDuration(TimeSpan amount)
```

Comment:
Probably better to merge `IncreaseDuration()` and `ExtendDuration` into a single method.

Reply:
I agree this is somewhat confusing and maybe the description should have been a little more clear, but ExtendDuration has several features we don't want here, most particularly that it resets the updatetime on the buff. We probably don't want to constantly call onExtend here either.